pkg/store/storepb/prompb/samples.go

```diff
@@ -42,6 +42,23 @@ func SamplesFromPromqlPoints(samples ...promql.Point) ([]Sample, []Histogram) {
 	return floats, histograms
 }
 
+// SamplesFromPromqlSeries converts promql.Series to a slice of Sample.
```

Comment:
Convert to both samples and histograms. Would you mind updating the comment a little bit